pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -292,17 +290,23 @@ def simulate_and_vjp(
 # The set of supported observables.
 
 
-def stopping_condition(op: qml.operation.Operator) -> bool:
+def stopping_condition(op: Operator) -> bool:
     """A function that determines whether or not an operation is supported by ``lightning.qubit``."""
     return op.name in _operations
 
 
-def accepted_observables(obs: qml.operation.Operator) -> bool:
+def stopping_condition_shots(op: Operator) -> bool:
+    """A function that determines whether or not an operation is supported by ``lightning.qubit``
+    with finite shots."""
+    return op.name in _operations or isinstance(op, (MidMeasureMP, qml.ops.op_math.Conditional))
```

Comment:
Codecov is complaining about this line coverage. Is this one more of codecov craziness or do we need to improve test coverage?

Reply:
This is happening on other lines too, but I will only comment here, you know the drill.

Reply:
Indeed, this is codecov :P we tests these lines in `./tests/new_api/test_device.py` but don't know why codecov fails here.

Reply:
3 device tests are failing due to this change in CI. I'll fix them and hopefully these codecov warnings will be vanished :)

Reply:
@AmintorDusko CI tests are passed but these CodeCov issue are still here :(

Reply:
@AmintorDusko CI tests are passed but these CodeCov warnings are still there :(